thetaPC/dnd-one-shot
# chore(20137558): Implement prettier & eslint.

[Ticket 20137558](https://github.com/users/thetaPC/projects/1/views/1?pane=issue&itemId=20137558)

How to test:
1. Run project
2. Verify that the project runs with no issues
